README.md

```diff
@@ -45,6 +34,20 @@ to developers/contributors.
 - [X] Implement Crypto
 - [X] Create Messenger
 
+## Q&A:
```

Comment:
It would fit very good into a FAQ section, because it's some common misconception about Tox, but I agree, it doesn't have to be in this PR.